ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -23,9 +23,9 @@
                 }
                 <nhs-form-group nhs-form-group-type=@IdFilterGroupState>
                     <label nhs-label-type="Standard"> ID number </label>
-                    <span class="nhsuk-hint " id="example-hint">
+                    <nhs-hint nhs-hint-type="Standard" id="id-types-hint">

```

Comment:
Much better `id` value, but we still want it to be connected to something. You'll want to add `aria-describedby="id-types-hint"` to the input (see https://github.com/nhsuk/frontend-dotnetcore/tree/master/src/NHSUKFrontEndLibraryTagHelpers/NHSUK.FrontEndLibrary.TagHelpers/Tags/Input#input-with-hint-text)
(ditto the others)